docs/src/main/sphinx/release/release-449.md

```diff
@@ -49,15 +50,17 @@
 
 ## Hudi connector
 
-* Add support for configuring the query retry policy on the S3 filesystem with
-  the `s3.retry-mode` and `s3.max-error-retries` configuration properties.
+* Add support for configuring the query retry policy on the native S3 filesystem
+  with the `s3.retry-mode` and `s3.max-error-retries` configuration properties.
+  ({issue}`21900`)
 * Improve performance of writing to Parquet files. ({issue}`22089`)
```

Comment:
Should we remove this line? Hudi connector doesn't support write operations.

Reply:
Removed